statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -32,6 +37,12 @@ func (psh *PresenterStatusHandler) GetCountConsensus() uint64 {
 	return psh.getFromCacheAsUint64(core.MetricCountConsensus)
 }
 
+// GetCountConsensusAcceptedBlocks will return count of how many times node was in consensus group and
```

Comment:
Small rephrase: "will return **a** count if how many times **the** node was ..."

Reply:
Done.